src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/custom.py

```diff
@@ -60,7 +87,7 @@ def cli_redis_create(client, resource_group_name, name, location, sku_name, # py
     """
     params = RedisCreateOrUpdateParameters(
         location,
-        Sku(sku_name, sku_family, sku_capacity),
+        Sku(sku, vm_size[0], vm_size[1]),
```

Comment:
I would recommend vm_size[1:] in case you ever support double digit capacity values.